class-sc-wp-updates-notifier.php

```diff
@@ -534,15 +579,22 @@ private function sort_by_interval( $a, $b ) {
 		 * much straight forward use of the WordPress Settings API.
 		 */
 		public function admin_settings_menu() {
-			$page = add_options_page( 'Updates Notifier', 'Updates Notifier', 'manage_options', 'wp-updates-notifier', array( $this, 'settings_page' ) );
-			add_action( "admin_print_scripts-{$page}", array( $this, 'enqueue_plugin_script' ) );
-		}
-
-		public function enqueue_plugin_script() {
-			wp_enqueue_script( 'wp_updates_monitor_js_function' );
+			add_options_page( 'Updates Notifier', 'Updates Notifier', 'manage_options', 'wp-updates-notifier', array( $this, 'settings_page' ) );
 		}
 
 		public function settings_page() {
+			// Trigger tests if they are ready to be sent.
+			$sc_wpun_send_test_slack = get_transient( 'sc_wpun_send_test_slack' );
+			if ( isset( $sc_wpun_send_test_slack ) && $sc_wpun_send_test_slack ) {
```

Comment:
isset is unnecessary here... it will be set, although possibly set to false if it doesn't exist.